includes/class-contact.php

```diff
@@ -135,7 +135,11 @@ public function widget( $args, $instance ) {
 
 		if ( 'yes' === $instance['map']['value'] && ! empty( $fields['address']['value'] ) ) {
 
-			add_action( 'wp_footer', [ $this, 'defer_map_iframes_js' ] );
+			if ( $this->defer_map_iframes && ! has_action( 'wp_footer', [ $this, 'defer_map_iframes_js' ] ) ) {
```

Comment:
@jonathanbardo I implemented it this way first, but in my testing it only printed the script once even when multiple maps were on the page.